uptime-kuma.json

```
@@ -293,72 +533,296 @@
+          "includeNullMetadata": true,
+          "legendFormat": "{{monitor_name}}",
+          "range": true,
+          "refId": "A",
+          "useBackend": false
+        }
+      ],
+      "title": "Response Times (ping based)",
+      "type": "timeseries"
     }
   ],
-  "schemaVersion": 30,
-  "style": "dark",
-  "tags": [],
+  "refresh": "1m",
+  "schemaVersion": 39,
+  "tags": [
+    "uptime-kuma",
+    "metrics"
+  ],
   "templating": {
-    "list": []
+    "list": [
+      {
+        "current": {
+          "selected": false,
+          "text": "Prometheus",
+          "value": "SUpEVGqnk"
+        },
+        "hide": 0,
+        "includeAll": false,
+        "label": "Datasource",
+        "multi": false,
+        "name": "datasource",
+        "options": [],
+        "query": "prometheus",
+        "queryValue": "",
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "type": "datasource"
+      },
+      {
+        "current": {
+          "selected": false,
+          "text": "Uptime-Kuma",
+          "value": "Uptime-Kuma"
+        },
+        "datasource": {
+          "type": "prometheus",
+          "uid": "${datasource}"
+        },
+        "definition": "label_values(job)",
+        "hide": 0,
+        "includeAll": false,
+        "label": "Job",
+        "multi": false,
+        "name": "job",
+        "options": [],
+        "query": {
+          "qryType": 1,
+          "query": "label_values(job)",
+          "refId": "PrometheusVariableQueryEditor-VariableQuery"
+        },
+        "refresh": 1,
+        "regex": "",
+        "skipUrlSync": false,
+        "sort": 1,
+        "type": "query"
+      },
+      {
+        "current": {
+          "selected": false,
+          "text": "status_online:3001",
+          "value": "status_online:3001"
+        },
+        "datasource": {
+          "type": "prometheus",
+          "uid": "SUpEVGqnk"
+        },
+        "definition": "query_result(uptime_status)",
+        "hide": 0,
+        "includeAll": false,
+        "multi": false,
+        "name": "Instance",
+        "options": [],
+        "query": {
+          "qryType": 3,
+          "query": "query_result(uptime_status)",
+          "refId": "PrometheusVariableQueryEditor-VariableQuery"
+        },
+        "refresh": 1,
+        "regex": "/.*instance=\"([^\"]*).*/",
+        "skipUrlSync": false,
+        "sort": 1,
+        "type": "query"
+      }
+    ]
   },
   "time": {
-    "from": "now-15m",
+    "from": "now-1h",
     "to": "now"
   },
-  "timepicker": {},
+  "timepicker": {
+    "refresh_intervals": [
+      "1m",
+      "5m",
+      "15m",
+      "30m"
+    ]
+  },
   "timezone": "",
-  "title": "Uptime Kuma",
-  "uid": "CN8E-vZ7k",
-  "version": 1,
-  "description": "A dashboard to show the data from the excellent Uptime Kuma project!"
-}
\ No newline at end of file
+  "title": "Uptime-Kuma Status",
+  "uid": "canLN3c4z",
+  "version": 31,
+  "weekStart": ""
```

```suggestion
  "description": "A simple dashboard showcasing service status updates over time tracked by the Uptime Kuma monitoring tool."
```

---

here you removed `weekStart`.
I am assuming this is not intentional:

```suggestion
  "weekStart": "",
  "description": "A simple dashboard showcasing service status updates over time tracked by the Uptime Kuma monitoring tool."
```